lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -205,6 +205,11 @@ def store(payment, options = {})
           return bank_token_response unless bank_token_response.success?
 
           params = { source: bank_token_response.params['token']['id'] }
+        elsif payment.is_a?(ActiveMerchant::Billing::NetworkTokenizationCreditCard)
+          tokenize_apple_google = tokenize_apple_google(payment, options)
+          return tokenize_apple_google unless tokenize_apple_google.success?
+
+          params = { source: tokenize_apple_google.params['token']['id'] }
```

Comment:
Would these changes also enable support for storing AP/GP via the base Stripe gateway?

Reply:
yes